rktlet/runtime/helpers.go

```diff
@@ -246,7 +246,10 @@ func generateAppAddCommand(req *runtimeApi.CreateContainerRequest, imageID strin
 
 			// Add uid, addtional gids.
 			if uid := secContext.RunAsUser; uid != nil {
-				cmd = append(cmd, fmt.Sprintf("--user=%s", *uid))
+				cmd = append(cmd, fmt.Sprintf("--user=%d", *uid))
+			}
+			if uname := secContext.RunAsUsername; uname != nil {
+				cmd = append(cmd, fmt.Sprintf("--user=%s", *uname))
```

Comment:
Do we have any escaping in place for those free-form `%s` we receive from the outside?